docs/signer.md

```diff
@@ -0,0 +1,354 @@
+# Kritis Signer
+
+Kritis Signer is a command-line tool that creates attestations. These attestations can be used for Kritis and Binary Authorization deployment enforcement.
```

Comment:
Kritis Signer is a command-line tool that creates attestations for a container image.

Reply:
Done.